### PR TITLE
Formatted code using EditorConfig

### PR DESCRIPTION
**Pull request by josteinaj**
Follow-up from #507.

### CODE REVIEW

File: src/main/resources/xml/xslt/split-html.xsl [hunk 1]
```diff
@@ -33,7 +33,7 @@
                     <xsl:copy-of select="/*/@* | @xml:base" exclude-result-prefixes="#all"/>
                     <xsl:namespace name="epub" select="'http://www.idpf.org/2007/ops'"/>
                     <xsl:text>
-</xsl:text>
+                    </xsl:text>
```
Comment (egli) [inline src/main/resources/xml/xslt/split-html.xsl:36]: Presumably this is not the same as before. The old code inserts a newline and no spaces. The new code inserts a newline and some spaces.

I tried the test suite and it passes. Are these additional spaces harmless?

**Review by egli — COMMENTED** [review]
Generally looks good, except for the comment about the additional spaces after the newline inside <xsl:text>

**Review by kalaspuffar — APPROVED** [review]

File: src/main/resources/xml/xslt/split-html.xsl (hunk 1 above)
Comment (josteinaj) [reply src/main/resources/xml/xslt/split-html.xsl:36]: Good point. It seems harmless, but the intention is to only insert a newline, so we should fix this. Maybe we could add a `xml:space="preserve"` on the `xsl:text` if EditorConfig respects that. Alternatively we could say `<xsl:text>&#x0A;</xsl:text>` or `<xsl:value-of select="'\n'"/>`or similar.

File: src/test/resources/2015-1/DTBook.invalid/C00000.invalid.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
     public static void main(String[] args) {
         ...
     }
-</code></p>
+                </code></p>
```
Comment (josteinaj) [inline src/test/resources/2015-1/DTBook.invalid/C00000.invalid.xml:47]: same newline issue here, we could maybe try either `xml:space="preserve"` on the `<code>` or `}&#x0A;</code>`

File: src/test/xprocspec/2015-1/dtbook-to-html/resources/DTBook/C00000.xml [hunk 3]
```diff
@@ -510,7 +510,7 @@
     public static void main(String[] args) {
         ...
     }
-</code></p>
+                </code></p>
```
Comment (josteinaj) [inline src/test/xprocspec/2015-1/dtbook-to-html/resources/DTBook/C00000.xml:513]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl (hunk 1 above)
Comment (josteinaj) [reply src/main/resources/xml/xslt/split-html.xsl:36]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl [hunk 4]
```diff
@@ -54,23 +54,23 @@
                         </xsl:if>
                     </head>
                     <xsl:text>
-</xsl:text>
+                    </xsl:text>
```
Comment (josteinaj) [inline src/main/resources/xml/xslt/split-html.xsl:57]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl [hunk 5]
```diff
@@ -54,23 +54,23 @@
...
-</xsl:text>
+                        </xsl:text>
                     </body>
                     <xsl:text>
-</xsl:text>
+                    </xsl:text>
                 </html>
                 <xsl:text>
-</xsl:text>
+                </xsl:text>
```
Comment (josteinaj) [inline src/main/resources/xml/xslt/split-html.xsl:73]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl [hunk 6]
```diff
@@ -54,23 +54,23 @@
...
-</xsl:text>
+                        </xsl:text>
                         <xsl:apply-templates select="$body/*"/>
                         <xsl:text>
-</xsl:text>
+                        </xsl:text>
                     </body>
                     <xsl:text>
-</xsl:text>
+                    </xsl:text>
```
Comment (josteinaj) [inline src/main/resources/xml/xslt/split-html.xsl:70]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl [hunk 7]
```diff
@@ -54,23 +54,23 @@
...
                         <xsl:if test="$body[self::header]">
                             <xsl:attribute name="epub:type" select="string-join(('frontmatter','titlepage',tokenize($body/@epub:type,'\s+')),' ')"/>
                         </xsl:if>
                         <xsl:text>
-</xsl:text>
+                        </xsl:text>
                         <xsl:apply-templates select="$body/*"/>
                         <xsl:text>
-</xsl:text>
+                        </xsl:text>
```
Comment (josteinaj) [inline src/main/resources/xml/xslt/split-html.xsl:67]: newline issue here

File: src/main/resources/xml/xslt/split-html.xsl [hunk 8]
```diff
@@ -54,23 +54,23 @@
...
-</xsl:text>
+                    </xsl:text>
                     <body>
                         <xsl:apply-templates select="$body/(@* except @xml:base)"/>
                         <xsl:if test="$body[self::header]">
                             <xsl:attribute name="epub:type" select="string-join(('frontmatter','titlepage',tokenize($body/@epub:type,'\s+')),' ')"/>
                         </xsl:if>
                         <xsl:text>
-</xsl:text>
+                        </xsl:text>
```
Comment (josteinaj) [inline src/main/resources/xml/xslt/split-html.xsl:64]: newline issue here

File: src/test/resources/2015-1/DTBook.invalid/C00000.invalid.xml (hunk 2 above)
Comment (josteinaj) [reply src/test/resources/2015-1/DTBook.invalid/C00000.invalid.xml:47]: newline issue here

File: src/test/resources/2015-1/DTBook/C00000.xml [hunk 9]
```diff
@@ -563,7 +563,7 @@
     public static void main(String[] args) {
         ...
     }
-</code></p>
+                </code></p>
```
Comment (josteinaj) [inline src/test/resources/2015-1/DTBook/C00000.xml:566]: newline issue here

**Review by josteinaj — COMMENTED** [review]
added "newline issue here" to lines where I could spot a newline issue similar to the one @egli pointed out.

File: src/main/resources/xml/xslt/split-html.xsl (hunk 1 above)
Comment (egli) [reply src/main/resources/xml/xslt/split-html.xsl:36]: > add a `xml:space="preserve"` on the `xsl:text` if EditorConfig respects that

I doubt EditorConfig is that smart :-)

> Alternatively we could say `<xsl:text>&#x0A;</xsl:text>`

That seems like the most straight forward solution

**Review by erikmtm — APPROVED** [review]

**Comment by josteinaj** [comment]
merged into `master` in d15c4efd15fd9c0743dd35201c4350d92dd2f887 :crossed_fingers:

**Pull request closed without merging**